src/lessons.json

```diff
@@ -37,5 +37,16 @@
     "lessonName": "Stacks & Queues",
     "lessonGroup": "Intro Data Structures",
     "files": []
+  },
+  "intro-to-cpp": {
+    "lessonName": "Intro to C++",
+    "lessonGroup": "Intro to Competitive Programming",
+    "files": [
+      "intro.mdx",
+      "input-output.mdx",
+      "io-fitbq.mdx",
```

Comment:
super nit: can we follow existing convention and say `io-fitb.mdx` instead? (without the q)